app/app.go

```diff
@@ -509,35 +513,35 @@ func New(
 
 	// Create Ethermint keepers
 	tracer := cast.ToString(appOpts.Get(srvflags.EVMTracer))
-	feeSs := app.GetSubspace(feemarkettypes.ModuleName)
-	app.FeeMarketKeeper = feemarketkeeper.NewKeeper(
-		appCodec,
-		runtime.NewKVStoreService(keys[feemarkettypes.StoreKey]),
-		authtypes.NewModuleAddress(govtypes.ModuleName),
-		keys[feemarkettypes.StoreKey], tKeys[feemarkettypes.TransientKey])
 
-	evmSs := app.GetSubspace(evmtypes.ModuleName)
+	app.FeeMarketKeeper = feemarketkeeper.NewKeeper(
+		appCodec, authtypes.NewModuleAddress(govtypes.ModuleName),
+		keys[feemarkettypes.StoreKey],
+		tKeys[feemarkettypes.TransientKey],
+	)
 
 	app.EvmKeeper = evmkeeper.NewKeeper(
 		appCodec,
-		runtime.NewKVStoreService(keys[evmtypes.StoreKey]),
 		keys[evmtypes.StoreKey],
 		tKeys[evmtypes.TransientKey],
 		authtypes.NewModuleAddress(govtypes.ModuleName),
 		app.AccountKeeper,
 		app.BankKeeper,
 		app.StakingKeeper,
-		&app.FeeMarketKeeper,
+		app.FeeMarketKeeper,
+		&app.Erc20Keeper,
 		tracer,
-		precompiles.StatefulContracts(
-			&app.FungibleKeeper,
-			app.StakingKeeper,
-			app.BankKeeper,
-			app.DistrKeeper,
-			appCodec,
-			storetypes.TransientGasConfig(),
-		),
-		aggregateAllKeys(keys, tKeys, memKeys),
+	)
+
+	app.Erc20Keeper = erc20keeper.NewKeeper(
```

Comment:
Do we need to define this one? I think this is the module to convert IBC tokens to ERC20? We would not use it for now as we're not IBC